authenticity/e-critical.yml

```diff
@@ -1,16 +1,16 @@
-openapi: 3.0.3
 components:
   schemas:
-
     Critical:
-      description: "Enumeration contains identifiers determining the criticality of the security element"
-      type: integer
+      description: Enumeration contains identifiers determining the criticality of
+        the security element
```

Comment:
description should be in one line, no need to add line separators.
in all files!